### ISSUE TITLE
Add notebook:comment-cell command

### ISSUE THREAD

**Issue by ianhi** [jupyterlab/jupyterlab#8743]
### Problem
When experimenting with code to analyze data I often find that I want to comment out an entire cell as it may not be the correct approach and I do not want to accidentally run it. However, I would prefer to not delete it if I think I may want return to it. Folding the cell is nice as it is no locally visually disruptive but I can still accidentally run it when I press `shift-enter` multiple times which is problematic if it overwrites important variables or is long running. So I find myself doing the following:
1. Navigating in jupyter command mode
2. `enter` to enter edit mode on the offending cell
3. `select-all` 
4. `crtl-/`
5. back to jupyter command mode
6. hide cell via keyboard shortcut

Instead I would like to be able to comment out the cell from jupyter command mode.

### Proposed Solution
I propose a command in the `notebook` command namespace that automates steps 2-3-4 in the problem statement. Then I could bind this to a keyboard shortcut of my choice.

Alternatively/additionally the ability to mark a cell as `Do not execute` or something similar would be great. 

### Additional context
Q: Is this irresponsible coding practice? A: Maybe, but it is a workflow I have naturally developed and would wager others have as well. Plus it doesn't seem that bad. It's sort of equivalent to having a short term storage shelf of cells.

Gif of my current approach:
![Peek 2020-07-29 20-58](https://user-images.githubusercontent.com/10111092/88868480-48a5ed00-d1de-11ea-9ed1-23fdbdfec2f1.gif)

**Comment by ianhi** [comment]
> It's sort of equivalent to having a short term storage shelf of cells.

Does anyone know if there is an extension that provides this? That would be an awesome extension.

**Comment by fcollonval** [comment]
Tip: in such case, I convert the cell to raw type. In command mode, the shortcut is `R` and `Y` if you want to go back to code cell.

**Comment by aiqc** [comment]
Agreed, just found myself commenting out a bunch of cells for the umpteenth time.

Since <kbd>CMD</kbd>+<kbd>/</kbd> is commonly used in IDEs for multi-line comments, perhaps `/` could be used as a way to comment out selected cells
